lang/src/org/partiql/lang/eval/ExprValueType.kt

```diff
@@ -37,70 +35,70 @@ enum class ExprValueType(val typeNames: List<String>,
                          val isSequence: Boolean = false,
                          val isRangedFrom: Boolean = false) {
     MISSING(
-        typeNames = listOf("missing"),
-        isUnknown = true
+            typeNames = listOf("missing"),
```

Comment:
nit (may be fixed by ktlint in the future): extra whitespace in these core type definitions, L113-116, L139-145, L151-184

Reply:
Fixed.